recipes/zlib/bld.bat

```diff
@@ -0,0 +1,24 @@
+:: Configure step.
+cmake -G "NMake Makefiles" ^
+      -D CMAKE_BUILD_TYPE=Release ^
+      -D CMAKE_PREFIX_PATH=%LIBRARY_PREFIX% ^
+      - DCMAKE_INSTALL_PREFIX:%LIBRARY_PREFIX% %SRC_DIR% ^
+      %SRC_DIR%
+if errorlevel 1 exit 1
+
+:: Build step.
+cmake --build %SRC_DIR% --target INSTALL --config Release
+if errorlevel 1 exit 1
+
+:: cmake Test.
+ctest
+if errorlevel 1 exit 1
```

Comment:
Yes, it is. Would be nice if there was a `set -e` strategy for Windows. If there is one, I don't know it. Anyways, LGTM and AppVeyor is running the test available.